features/prompt_memes_a/challenge_promptmeme_setup.feature

```diff
@@ -181,6 +181,8 @@ Feature: Prompt Meme Challenge
   When I follow "Edit Prompt"
   Then I should see single prompt editing
   And I should see "Edit Sign-up"
+  When I change my Battle 12 signup to invalid combination A2
+  Then I should see errors
```

Comment:
I think it's better to be specific what changes we make and what exact errors we get as a result. So instead of adding a new step, you could write:

```gherkin
When I uncheck "Stargate Atlantis"
  And I press "Submit"
Then I should see "<the specific error message>"
```

Reply:
And done. Thank you for talking me through this.

Reply:
No problem! Looks good, just one last change, can you add to the check step?
```gherkin
Then I should see "Sorry! We couldn't save this request because:"
  And I should see "Your Request must include exactly 1 fandom tags"
```

Reply:
Done!